src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+const EnterpriseOffersBalanceAlert = ({
```

Comment:
refactored this: because there are only 2 choices right now and it's contingent upon hasNoEnterpriseOfferBalance, I decided to hand the component the bool and then let it generate the appropriate values based on that (which also removes the need for the separate utils function.